docs/core/tools/csproj.md

```diff
@@ -235,6 +235,12 @@ Specifies the URL for the repository where the source code for the package resid
 ### RepositoryType
 Specifies the type of the repository. Default is "git". 
 
+### RepositoryBranch
+Specifies the name of the source branch in the repository. When the project is packaged in a Nuget package it is added to the package metadata.
```

Comment:
```suggestion
Specifies the name of the source branch in the repository. When the project is packaged in a NuGet package it is added to the package metadata.
```
or 
```suggestion
Specifies the name of the source branch in the repository. When the project is packaged in a nuget package it is added to the package metadata.
```

In general NuGet or nuget, rather than Nuget. 

I see the pattern in this page seems to be NuGet so  I'd suggest that.